Library/Formula/nsd.rb

```diff
@@ -24,12 +22,16 @@ def install
       --with-libevent=#{Formula["libevent"].opt_prefix}
     ]
 
-    args << '--enable-root-server' if build.with? 'root-server'
-    args << '--enable-bind8-stats' if build.with? 'bind8-stats'
-    args << '--enable-ratelimit' if build.with? 'ratelimit'
-    args << '--enable-zone-stats' if build.with? 'zone-stats'
+    args << "--enable-root-server" if build.with? "root-server"
+    args << "--enable-bind8-stats" if build.with? "bind8-stats"
+    args << "--enable-ratelimit" if build.with? "ratelimit"
+    args << "--enable-zone-stats" if build.with? "zone-stats"
 
     system "./configure", *args
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
+    system "#{sbin}/nsd", "-v"
```

Comment:
The "-v" option prints the version. nsd doesn't have long options. Other than starting the server, I can't think of any other way to test it. I guess that printing the version will have to do for now.